src/auth/project-config.ts

```diff
@@ -111,6 +137,9 @@ export class ProjectConfig {
     if (typeof response.smsRegionConfig !== 'undefined') {
       this.smsRegionConfig = response.smsRegionConfig;
     }
+    if (typeof response.mfa !== 'undefined') {
```

Comment:
Super nit: Add a space "/** Backend..." and also no single quote at the end of the comment string.

Reply:
I think a comment like `'Backend API returns "mfa" in case of project config and "multiFactorConfig" in case of tenant config. The SDK exposes it as multiFactorConfig always.'` would be useful. @pragatimodi